frontend/test/metabase/scenarios/question/new.cy.spec.js

```diff
@@ -160,7 +160,7 @@ describe("scenarios > question > new", () => {
             .click();
         });
       // this step is maybe redundant since it fails to even find "by month"
-      cy.findByText("Hour of day");
+      cy.findByText("Hour of Day");
```

Comment:
The base branch seems to be passing. What changed here?

Reply:
With the change in this PR, the temporal subdimensions came from a different source. They are still the same but somehow the display names are different. I'm looking at that (as a separate issue). This was also observed in previous PR #15530.